Gemfile

```diff
@@ -1,14 +1,14 @@
 source 'https://rubygems.org'
 
-ruby '2.6.1'
+ruby '2.6.4'
 gem 'rails', '~> 5.1.6.1'
 
 # Use postgresql as the database for Active Record
-gem 'pg', '~> 1.1'
+gem 'pg', '~> 1.1.0'
 # Use Puma as the app server
-gem 'puma', '~> 3.12'
+gem 'puma', '~> 3.12.0'
 
-gem 'therubyracer', platforms: :ruby
+gem 'mini_racer'
```

Comment:
:eyes: Can you tag this gem version :pray:  in Gemfile.lock says we are using `0.2.6`